### DISCUSSION TITLE
Context provided to the model when using "inline assist"?

### DISCUSSION THREAD

**Discussion by osa1** [zed-industries/zed#17631] [Feature Requests]
When using "inline assist" what is the context provided to the AI model?

From my testing it looks like it's `/file`, but wanted to be sure.

Is it possible to provide `/project` as the context?

Is it planned to have AI do project-wide changes (instead of just in the selected region in the current file), or is it too ambitious?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
